packages/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/types/FlutterSignInRequest.kt

```diff
@@ -23,7 +23,16 @@ import com.amplifyframework.core.Amplify
 data class FlutterSignInRequest(val map: HashMap<String, *>) {
   val username: String = map["username"] as String;
   val password: String = map["password"] as String;
-  val options: HashMap<String, *>? = map["options"] as HashMap<String, *>?;
```

Comment:
I think we need to import `AWSCognitoAuthSignInOptions`

Reply:
I noticed you guys have these classes to deal with options when I was writing the swift code. I'm trying to fix the iOS side that still is not working for me.

Reply:
I'm working on it too (I pulled your branch).  I'm going to be suggesting some revisons to this PR to make sure the options can be null when they come over the platform channel.  Thanks so much for submitting this.